app/core/config.py

```diff
@@ -7,8 +7,8 @@
 
 class Settings(BaseSettings):
     main_path: str = os.path.abspath(os.path.dirname(__file__))
-    models_path: str = os.path.join(main_path, "labels")
-    routers_path: str = os.path.join(main_path, "routers")
+    models_path: str = os.path.join(main_path, "../labels")
+    routers_path: str = os.path.join(main_path, "../routers")
 
     POSTGRES_SERVER: str
```

Comment:
I'd recommend also including an env variable for port, say `POSTGRES_PORT`. Update env-example too

Reply:
Should we docker changes in a separate task? Up to you.